src/MSI/Product.wxs

```diff
@@ -22,6 +22,7 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
     <Property Id="WIXUI_INSTALLDIR" Value="INSTALLFOLDER"/>
 
     <WixVariable Id="WixUIDialogBmp" Value="Resources\DialogBackground.png" />
+    <WixVariable Id="WixUIBannerBmp" Value="Resources\WixDialogBanner.png" />
```

Comment:
Can we not use the existing BrandIcon_256x.png file? Would be good to keep down the number of copies of our icons.

Reply:
Dimensions don't match unfortunately - please see #292 for more info. I am looking into scaling options w/ Fer and Chelsea ATM